docs/authentication.md

```diff
@@ -0,0 +1,91 @@
+# Authentication
+
+## Form Types
+There are four types of forms related to authentication. The only one not located in Reaction is Reset Password, which currently exists as a coffeescript template in Force.
+
+### Login
+There are three ways users can login to Artsy.
+  1. Email
+  2. Facebook
+  3. Twitter (signups deprecated)
+
+### Signup
+There are two ways users can signup to Artsy. Twitter signups have been deprecated for a while and we intend to convert existing Twitter users to email users eventually.
+  1. Email
+  2. Facebook
+
+New signups require a name, email, 8 character password minimum, and they must accept our terms, policies and to receive emails.
+
+### Forgot Password
+If a user forgets their password, they are able to reset it using the Forgot Password form. This form only accepts an email address. Reset instructions will be sent to the user along with a reset token which is used to verify the user.
+
+Note: In Reaction, we incorrectly use the name ResetPassword.
+
+### Reset Password
+This is the only form that exists as a coffeescript template in Force. To successfully reset your password, the form must be submitted with a valid token (typically taken from a query param).
+
+## Entry Points
+There are three ways a user can see authentication forms. Note that each of these are using the same underlying form infrastructure which includes validation, form submission, and analytics events.
+
+### Modal
+Authentication modals can only be triggered with the `mediator`. The `mediator` is a global event emitter/listener in Force that is responsible for various event handling. Below is an example of how a modal might be triggered.
+
+```js
+mediator.trigger("open:auth", {
+  mode: "signup",
+  copy: "Sign up to receive alerts when new works are available by artists you follow.",
+  signupIntent: "follow artist"
+})
+```
+
+### URL Route
+We can also go to `/login`, `/signup`, and `/forgot` directly to access authentication forms.
+
+### Embed
+Similar to the above, authentication forms can be embedded anywhere on the site. (TODO: Add Artist CTA example once implemented)
+
+## Implementation
+### Basics
+We use [Formik](https://github.com/jaredpalmer/formik) to handle form entries and [Yup](https://github.com/jquense/yup) for form validation. Mobile uses the `Wizard` component to handle the multi-step flow. The form submission is handled in Force, and the authentication backend is handled in [artsy-passport](https://github.com/artsy/artsy-passport).
+
+### Component Responsibilities
+**Desktop**
+- `ModalContainer` (Force): Listens for changes to Force's mediator and triggers opens to ModalManager. It is also responsible for setting Cookies like `destination` and `afterAuthAction`.
```

Comment:
Can we link to these components directly?

Reply:
It would also be good to link to the associated StoryBooks entry

Reply:
Just linked the Reaction stuff for now since we're going to swap auth2 -> auth soon in Force. I'll link it once that work is done!